christfellowshipchurch/web-app-v2
# Site searched track event segment

### About
This PR allows for Segment to track details when a user searches on the discover page 

### Test Instructions
Open the test link
Open the Discover Page
Search for something and click on "search"
Open the Segment Event Tracker Chrome extension
You should see something like the example under Screenshots

The properties to look at are:
search_term
number_of_results
search_type

### Screenshots
<img width="1436" alt="Screen Shot 2023-01-25 at 2 35 57 PM" src="https://user-images.githubusercontent.com/46769629/214672695-484301da-2169-4146-8f39-a14dd2f7a134.png">

### Closes Tickets
[CFDP-2387](https://christfellowshipchurch.atlassian.net/jira/software/projects/CFDP/boards/2?selectedIssue=CFDP-2387)



[CFDP-2387]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2387?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
